### DISCUSSION TITLE
Enterprise git commit counts do not match external git commit counts

### DISCUSSION THREAD

**Discussion by claytonjhamilton** [community/community#31249] [Enterprise]
Hello there!

# Issue

I've noticed that the public repo commits from my external GH account do not appear on my Enterprise GH account (note the recent weekend commits).

Here is my Enterprise GitHub account contribution graph:
![image](https://user-images.githubusercontent.com/51751663/187258387-24b6e2a2-738a-4869-8b73-1a38fdcf58cf.png)

And here is my GitHub account contribution graph:
![image](https://user-images.githubusercontent.com/51751663/187258607-94908168-f933-44a4-a79b-c745f4d07355.png)

## Things I've Tried

I've ensured that emails for my external GH and enterprise GH are associated with both accounts. I've "Updated contributions" on my enterprise public profile. Still no luck. 

In case it matters, we are on GitHub Enterprise Server 3.1.14

Any and all help is greatly appreciated!

**Comment by jsspen** [comment]
Did you ever get this figured out? I've been struggling with this, too. It seems crazy to think it's not possible as its such an obvious need but...

**Reply by claytonjhamilton** [reply]
No, I wish I had. I suspect that our enterprise server admins don't want external GH connecting to EC GH at all. Just a hunch.
